components/helpers/applyFilter.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable */
+export function sortFilter(arr: { value: string }[]): { value: string }[] {
```

Comment:
Kindly resolve the eslint errors instead of disabling them

Reply:
@vishvamsinh28 instead of disabling the eslint for whole file, try to resolve them by reading the eslint docs like - https://eslint.org/docs/latest/rules/no-restricted-syntax, https://eslint.org/docs/latest/rules/guard-for-in and make the required changes accordingly.